owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.kt

```diff
@@ -1060,53 +1062,115 @@ class FileDisplayActivity : FileActivity(),
         val context = this@FileDisplayActivity
         if (!uiResult.data.isNullOrEmpty()) {
             var pos = 0
-            data.asReversed().forEach { file ->
-                AlertDialog.Builder(context)
+            val fragmentManager = supportFragmentManager
+            data.asReversed().forEachIndexed { index, file ->
+                val countDownValue = index + 1
+
+                val customDialog = FileAlreadyExistsDialog.Builder()
                     .setTitle(
-                        if (file.isFolder) {
-                            R.string.folder_already_exists
-                        } else {
-                            R.string.file_already_exists
-                        }
+                        this.getString(
+                            if (file.isFolder) {
+                                R.string.folder_already_exists
+                            } else {
+                                R.string.file_already_exists
+                            }
+                        )
                     )
-                    .setMessage(
-                        String.format(
-                            context.getString(
-                                if (file.isFolder) {
-                                    R.string.folder_already_exists_description
-                                } else {
-                                    R.string.file_already_exists_description
-                                }
-                            ), file.fileName
+                    .setDescription(
+                        this.getString(
+                            if (file.isFolder) {
+                                R.string.folder_already_exists_description
+                            } else {
+                                R.string.folder_already_exists_description
+                            }, file.fileName
                         )
                     )
-                    .setNeutralButton(R.string.welcome_feature_skip_button) { _, _ ->
-                        replace.add(null)
-                        pos++
-                        if (pos == data.size) {
-                            launchAction(uiResult.data, replace)
+                    .setCheckboxText(this.getString(R.string.apply_to_all_conflicts, countDownValue.toString()))
+                    .build()
+
+
+                if (countDownValue > 1) {
+                    customDialog.checkboxVisible = true
+                }
+
+                customDialog.show(fragmentManager!!, "customdialog")
```

Comment:
I will probably add a let here, because if fragmentManager is null it will crash the app.

Reply:
Done